tests/test_regression.py

```diff
@@ -4,40 +4,48 @@
 from pathlib import Path
 from typing import Any
 
+import pytest
+
 from autocorpus.configs.default_config import DefaultConfig
 
 
-def test_autocorpus(data_path: Path):
-    """A regression test for the main Auto-CORPus class on the Auto-CORPus paper.
+@pytest.mark.parametrize(
+    "input_file, config",
+    [
+        ("PMC/Pre-Oct-2024/PMC8885717.html", DefaultConfig.LEGACY_PMC.load_config()),
+        ("PMC/Current/PMC8885717.html", DefaultConfig.PMC.load_config()),
+    ],
+)
+def test_autocorpus(data_path: Path, input_file: Path, config: dict[str, Any]) -> None:
```

Comment:
This is technically a str
```suggestion
def test_autocorpus(data_path: Path, input_file: str, config: dict[str, Any]) -> None:
```

Reply:
Good spot, corrected